testbed/eks/lib/addons/fluxv2.ts

```diff
@@ -0,0 +1,110 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import * as yaml from 'js-yaml';
+import * as request from 'sync-request';
+
+export interface FluxV2Props extends cdk.StackProps {
+    cluster: eks.Cluster;
+    fluxRepos: Repository[];
+    secretName: string;
+    fluxVersion: string;
+}
+export class Repository {
+    repoUrl: string;
+    repoBranch: string;
+    repoPath: string;
+
+    constructor(repoUrl: string, repoBranch?: string, repoPath?: string) {
+        if (!repoBranch) {
+            this.repoBranch = "main"
+        } else {
+            this.repoBranch = repoBranch
+        }
+        if (!repoPath) {
+            this.repoPath = "test/workflows"
+        } else {
+            this.repoPath = repoPath
+        }
+        this.repoUrl = repoUrl
+    }
+}
+class FluxRelease {
```

Comment:
I'm not sure this FluxRelease class is worth its weight. Then entire class can be boiled down into 
```
props.cluster.addManifest('flux', ...yaml.safeLoadAll(
  request('GET', `https://github.com/fluxcd/flux2/releases/download/${this.fluxVersion}/install.yaml).getBody()
))
```
Consider putting this directly into the Flux construct. This might be further simplified with using helm (though it looks like you might still need something like this to install the CRDs).